.github/workflows/dotnetcore.yml

```
@@ -0,0 +1,25 @@
+name: .NET Core
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with: 
+        fetch-depth: 0 # avoid shallow clone so nbgv can do its work.
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.100
```

Should we maybe build with the latest .NET 6 SDK? In that case we also want to schedule this workflow to run daily or weekly so we get notified of potential build failures when a new .NET 6 SDK is released.

```suggestion
        dotnet-version: 6.0.x
```

---

Yes this seems like an excellent idea.